src/etc/inc/interfaces.inc

```diff
@@ -4022,12 +4050,20 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 		killbypid("{$g['varrun_path']}/rtsold_{$wanif}.pid");
 		sleep(2);
 	}
+	if (isset($wancfg['dhcp6withoutra'])){
+		kill_dhcp6client_process($wanif);	
+		sleep(3);
```

Comment:
Is there any reason to sleep here? Seems that just adds unnecessary delay. There is already a sleep in kill_dhcp6client_process to give it time to die off, so it seems like it should be safe to move on immediately here without the sleep.